templates/sidebar/home_sidebar.html

```diff
@@ -34,17 +34,17 @@ <h4 class="card-title mb-0">RICOH THETA X</h4>
         <ul class="nav nav-sm nav-tabs nav-vertical mb-4">
           <li class="nav-item">
             <a class="nav-link active" href="{{url_for('info')}}">
-              <i class="bi-person-badge nav-icon"></i> Info
+              <i class="bi-info-square-fill nav-icon"></i> Info
             </a>
           </li>
           <li class="nav-item">
             <a class="nav-link" href="{{url_for('state')}}">
-              <i class="bi-shield-shaded nav-icon"></i> State
+              <i class="bi-clipboard2-data-fill nav-icon"></i> State
             </a>
           </li>
           <li class="nav-item">
             <a class="nav-link" href="{{url_for('take_picture')}}">
-              <i class="bi-bell nav-icon"></i> Shutter
+              <i class="bi-lightning-fill nav-icon"></i> Shutter
```

Comment:
The lightning looks like a camera flash.  Suggest you change it to a normal shutter button, like round circle. 

consider circle or circle-fill

![image](https://github.com/codetricity/theta-flask-desk/assets/5799860/11b9f197-74f9-4986-b720-cf8d8dcbdea7)

Reply:
OK, I basically agree with your suggestions. I will do a pull request with these updates.

Reply:
Added in the most recent pull request: https://github.com/codetricity/theta-flask-desk/pull/18/commits/5931eeae835d2bcd48a317ed36a19626207956c7